docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.4'
+
+services:
+  geta-socialchannels-sample:
+    image: ${DOCKER_REGISTRY-}geta-socialchannels-sample
+    build:
+      context: .\samples\Geta.SocialChannels.Sample
+      dockerfile: Dockerfile
+    networks: 
+     - geta-socialchannels-network
+    depends_on:
+     - geta-socialchannels-db
+   
```

Comment:
Can you add a static IP address like here: https://github.com/Geta/Tags/blob/master/samples/Geta.Tags.Sample/docker-compose.yml#L11

Use IP address: 172.16.238.21

Reply:
Change applied.